I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/minigame/trpg_combat/combat_boss_game.rpy", line 198, in script
    $ game.run()
  File "game/minigame/trpg_combat/combat_boss_game.rpy", line 198, in <module>
    $ game.run()
  File "game/minigame/trpg_combat/combat_boss_game.rpy", line 115, in run
    renpy.say("You found the secret escape button!")
TypeError: say() missing 1 required positional argument: 'what'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/minigame/trpg_combat/combat_boss_game.rpy", line 198, in script
    $ game.run()
  File "/Users/namseoa/Desktop/renpy-8.1.1-sdk/renpy/ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Users/namseoa/Desktop/renpy-8.1.1-sdk/renpy/python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/minigame/trpg_combat/combat_boss_game.rpy", line 198, in <module>
    $ game.run()
  File "game/minigame/trpg_combat/combat_boss_game.rpy", line 115, in run
    renpy.say("You found the secret escape button!")
TypeError: say() missing 1 required positional argument: 'what'

macOS-13.2-arm64-arm-64bit arm64
Ren'Py 8.1.2.23090503
A Trick Of The Mind 1.0
Wed Sep 27 18:03:21 2023
